src/bed-admission/active-patients/eligible-admissions.resource.ts

```diff
@@ -0,0 +1,43 @@
+import { useMemo } from "react";
+import useSWR from "swr";
+import { openmrsFetch } from "@openmrs/esm-framework";
+
+interface PatientAdmissionListResponse {
+  idNumber: string;
+  name: string;
+  age: number;
+  gender: string;
+  visitType: string;
+  visitStartTime: string;
+  admissionEncounterUuid: string;
+  patientUuid: string;
+}
+
+export function useEligibleAdmissions() {
+  const requestUrl = `rest/v1/kenyaemr/sql/?q=bedManagement.sqlGet.patientListForAdmission`;
```

Comment:
```suggestion
  const requestUrl = `rest/v1/kenyaemr/sql/?q=bedManagement.sqlGet.patientListForAdmission`;
```

should we make this endpoint configurable?